lib/viewer.coffee

```diff
@@ -99,7 +100,7 @@ class Viewer extends Disposable
     if @tabView? and atom.workspace.paneForItem(@tabView)?
       atom.workspace.paneForItem(@tabView).activateItem(@tabView)
     else
-      @tabView = new PDFView(@url)
+      @tabView = new PDFView(@url,"""#{path.basename(pdfPath)}""")
```

Comment:
Can you please change this to 
`@tabView = new PDFView(@url, path.basename(pdfPath))`